certora/scripts/Round1/verifyAll.sh

```diff
@@ -16,7 +18,7 @@ do
                 certoraRun certora/harnesses/ERC20VotesHarness.sol certora/harnesses/$contractFile \
                 --link ${contractFile%.*}:token=ERC20VotesHarness \
                 --verify ${contractFile%.*}:certora/specs/$specFile "$@" \
-                --solc solc8.2 \
+                --solc solc \
```

Comment:
Other scripts, such as `certora/scripts/Round1/sanity.sh` and `certora/scripts/Round1/sanityTokens.sh` uses `solc8.2`.
We should unify that.

Reply:
scripts ran by the ci downloads one version of solc and so uses `solc` executable. Some members on the team have a set of executables to make switching between versions easier. I can remove this script as its not ran by ci or leave it in case others from Certora, OZ, or the community try to add rules or run them.